client/blocks/login/index.jsx

```diff
@@ -392,6 +415,17 @@ class Login extends Component {
 			);
 		}
 
+		if ( this.showContinueAsUser() ) {
+			// someone is already logged in, offer to proceed to the app without a new login
+			return (
+				<ContinueAsUser
+					onChangeAccount={ () => {
```

Comment:
Should this be declared as a method instead of being redefined on each render?

Reply:
Yes. Thanks for pointing this out! Fixed in https://github.com/Automattic/wp-calypso/pull/40125/commits/3882399e392ca7daec22ab9dac4b07b82bdf3ec3